command/agent/profile/pprof.go

```diff
@@ -0,0 +1,127 @@
+package profile
```

Comment:
~ Should we call this package `pprof`?  `profile` seems quite overloaded here - to refer to cpu profile, generic lookup profile endpoint, and profile package?

Reply:
yes I like pprof better, changing